src/main/java/ch/wisv/areafiftylan/controller/AuthenticationController.java

```diff
@@ -61,6 +72,19 @@ public ModelAndView getLoginPage() {
         return createResponseEntity(HttpStatus.OK, "Here's your token!");
     }
 
+    @RequestMapping(value = "/token", method = RequestMethod.POST)
+    public ResponseEntity<?> createAuthenticationToken(@RequestBody UserDTO userDTO) {
+        String authToken = authenticationService.createNewAuthToken(userDTO.getUsername(), userDTO.getPassword());
+
+        return createResponseEntity(HttpStatus.OK, "Token successfully created", authToken);
+    }
+
+    @PreAuthorize("isAuthenticated()")
+    @RequestMapping(value = "/token/verify", method = RequestMethod.POST)
+    public ResponseEntity<?> verifyToken() {
```

Comment:
The token is checked in the Filter. If it's not valid, there won't be an Authenticated user so the `PreAuthorize` will catch it.